cluster/gce/util.sh

```diff
@@ -2720,6 +2732,15 @@ function create-master() {
       --allow "tcp:2380,tcp:2381" &
   fi
 
+  # Create rule for etcd grpc proxy.
+  if [[ "${START_ETCD_GRPC_PROXY:-}" == "true" ]]; then
+    gcloud compute firewall-rules create "${MASTER_NAME}-etcd-grpc-proxy" \
+      --project "${NETWORK_PROJECT}" \
+      --network "${NETWORK}" \
+      --source-ranges "0.0.0.0/0" \
+      --allow tcp:23790 &
```

Comment:
also 23790 here. it's spread in different files. it'd be great if we could get these in a common place.

Reply:
Actually, I did not use a variable for "23790" as I found throughout the script there is no variable for "2379", which is the port for core etcd server. Actually, the string "2379" is used much more frequently than "23790". I guess the reason is that it is a script, which is hard to debug and thus readability is more important.